spec/features/shared_examples/cherrypicking.rb

```diff
@@ -190,6 +190,11 @@
             end
           end
 
+          def get_machine_barcode_for_bed(bedcode)
+            num_barcode = robot.robot_properties.where(key: bedcode).first.value
+            SBCF::SangerBarcode.new(prefix: 'RB', number: num_barcode).machine_barcode
```

Comment:
Something is wrong with this (in the original code too). Bed barcodes are prefix 'BD', not 'RB' which is for robots.
So the bed verification validation should be failing because the barcode is wrong.

Reply:
Yes, that seems to be the verification process in the robot is not checking with the full barcode and prefix, but only with the number of the barcode. 
I've changed to BD and created a ticket in Genomic pipelines backlog to solve the problem with the robot verification. It's this issue: https://github.com/sanger/sequencescape/issues/2845